libsql-wal/src/segment/current.rs

```diff
@@ -261,7 +265,35 @@ impl<F> CurrentSegment<F> {
     pub fn tail(&self) -> &Arc<SegmentList<F>> {
         &self.tail
     }
+
+    // todo: maybe return boxed frames?
+    pub fn rev_frame_stream(&self) -> impl Stream<Item = Result<Frame>> + '_
+    where
+        F: FileExt,
+    {
+        async_stream::try_stream! {
+            let (start_frame_no, last_committed) = {
+                let header = self.header.lock();
+                dbg!((header.start_frame_no.get(), header.last_commited_frame_no.get()))
```

Comment:
dbg

Reply:
should we just convert these dbg into tracing::debug?

Reply:
will do as followup 👍